### PR TITLE
Xtap updates based on schema changes

### PR DESCRIPTION
**Pull request by jkdouglas**
Schema update PR: 5890457

### CODE REVIEW

**Comment by jkdouglas** [comment]
Corresponding schema update PR: https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5890457

**Comment by Lauragra** [comment]
@FaithOmbongi , can you please approve this one before I merge? Thanks.

File: api-reference/beta/resources/crosstenantaccesspolicy.md [hunk 1]
```diff
@@ -29,7 +29,6 @@ Inherits from [tenantRelationshipAccessPolicyBase](../resources/tenantrelationsh
 |Property|Type|Description|
 |:---|:---|:---|
 | displayName | String | The display name of the cross-tenant access policy. Inherited from [policyBase](../resources/policybase.md).|
-| lastModifiedDateTime | DateTimeOffset | The time that the cross tenant access policy was last modified represented using ISO 8601 format and always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/crosstenantaccesspolicy.md:32]: Please also delete this property from the JSON representations section.

File: changelog/Microsoft.DirectoryServices.json [hunk 2]
```diff
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "62535136-e63d-4727-ae28-84d7a9ed964a",
+          "ApiChange": "Resource",
+          "ChangedApiName": "crossTenantAccessPolicyConfigurationBase",
+          "ChangeType": "Removal",
+          "Description": "Removed the crossTenantAccessPolicyConfigurationBase resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:10]: Should be `Deleted`.
```suggestion
          "ChangeType": "Deletion",
          "Description": "Deleted the **crossTenantAccessPolicyConfigurationBase** resource.",
```

**Review by FaithOmbongi — APPROVED** [review]
Thank you @jkdouglas for raising this PR. I've approved with comments - requested for 2 minor fixes.

As a follow-up (non-blocking for this PR), please inspect why the payload from calling /policies/crossTenantAccessPolicy API is different from the schema. There's a new property **allowedCloudEndpoints**. Let's try to align the payloads with the schema.

File: api-reference/beta/resources/crosstenantaccesspolicy.md (hunk 1 above)
Comment (jkdouglas) [reply api-reference/beta/resources/crosstenantaccesspolicy.md:32]: @FaithOmbongi - allowedCloudEndpoints is a new property that will be documented as part of the public preview of cross-cloud b2b. Here is the PR I have for that: https://github.com/microsoftgraph/microsoft-graph-docs/pull/16378

**Pull request merged**